Gulpfile.mjs

```diff
@@ -8,13 +8,12 @@ import through from "through2";
 import chalk from "chalk";
 import newer from "gulp-newer";
 import babel from "gulp-babel";
-import camelCase from "lodash/camelCase.js";
 import fancyLog from "fancy-log";
 import filter from "gulp-filter";
 import revertPath from "gulp-revert-path";
 import gulp from "gulp";
 import { rollup } from "rollup";
-import { babel as rollupBabel } from "@rollup/plugin-babel";
+import _rollupBabel from "@rollup/plugin-babel";
```

Comment:
What is this change needed for? :thinking:

Reply:
Someone weird I had to do for windows? 

It's similar to what we did for "rollup-plugin-dts"?

Reply:
For `rollup-plugin-dts` it's because Node.js cannot recognize its exports, but for `@rollup/plugin-babel` it should work since it does `exports.babel = ...` :thinking: 

Maybe you have a very old Node.js version, from before that Node.js used `cjs-module-lexer`? Otherwise it seems like a Node.js bug.

Reply:
I'll paste the error if that helps? I'm on later node (`$ node -v v14.9.0`)

```
file:///C:/Users/Hen/win-dev/babel/gulpfile.mjs:17
import { babel as rollupBabel } from "@rollup/plugin-babel";
         ^^^^^
SyntaxError: The requested module '@rollup/plugin-babel' is expected to 
be of type CommonJS, which does not support named exports. CommonJS modules can be imported by importing the default export.
For example:
import pkg from '@rollup/plugin-babel';
const { babel: rollupBabel } = pkg;
```